lms/djangoapps/django_comment_client/management_utils.py

```diff
@@ -0,0 +1,50 @@
+"""
+This module provides utility functions for django_comment_client
+"""
+from pymongo.errors import PyMongoError
+from pymongo import MongoClient
+
+from django.conf import settings
+from django.contrib.auth.models import User
+
+MONGO_PARAMS = settings.FORUM_MONGO_PARAMS
+
+CREDENTIALS = ''
```

Comment:
This block here looks to be repeated below in our tests. Can we refactor it into a common helper method? Maybe:

```python
def get_mongo_connection_string():
    # ...
```